test/entity.test.js

```diff
@@ -47,22 +47,15 @@ describe('handles block entities', () => {
         const simplified = nbt.simplify(tag)
         sign.entity = tag
 
-        if (registry.type === 'pc') {
-          assert.deepEqual(simplified.Text1, JSON.stringify(sign.blockEntity.Text1.json))
-        }
-
-        if (registry.type === 'bedrock') {
-          assert.deepEqual(simplified.Text, sign.blockEntity.Text)
-        }
+        assert.deepEqual(simplified.Text, sign.blockEntity.Text)
```

Comment:
This should not be passing on 1.20

Reply:
Yeah, the test is to make sure the correct NBT is set when updating the sign text.

Reply:
Replaced it with a test that makes sure the NBT is set correctly in the right format